src/test/java/com/spotify/faceapi/Service/TopTrackofArtistTest.java

```diff
@@ -0,0 +1,122 @@
+
+@RunWith(MockitoJUnitRunner.class)
+public class TopTrackofArtistTest {
+
+    @InjectMocks
+    private TopTrackofArtist topTrackofArtist;
+
+    @Mock
+    private RestTemplate restTemplate;
+
+    private Logger logger = Logger.getLogger(TopTrackService.class.getName());
+
+    @Test
+    public void getTopTracksofArtist_Success() throws NoAccountException, ForbiddenException, NoArtistsException, BadRequestException, NotFoundException, UnAuthorizedException, NoTrackException, IOException, ParseException {
+
+        JSONParser parser = new JSONParser();
+        ObjectMapper mapper = new ObjectMapper();
+
+        Object json_obj3  = parser.parse(new FileReader("C:\\PROJECT\\NEW\\faceapi\\SEPM\\src\\test\\java\\com\\spotify\\faceapi\\data\\TopArtiststracks.json"));
+        Map<String, String> topArtiststracks_map1 = mapper.readValue(json_obj3.toString(), Map.class);
+
+        ResponseEntity responseEntity = new ResponseEntity(topArtiststracks_map1, HttpStatus.OK);
+        Mockito.when(restTemplate.exchange(ArgumentMatchers.anyString(),
+                ArgumentMatchers.any(HttpMethod.class),
+                ArgumentMatchers.<HttpEntity<?>> any(),
+                ArgumentMatchers.<Class<String>> any())).thenReturn(responseEntity);
+
+        LinkedHashMap res = (LinkedHashMap) topTrackofArtist.getTopTracksofArtist("AZT1010ZZ7H2","1mYsTxnqsietFxj1OgoGbG");
+        Assert.assertEquals(topArtiststracks_map1,res);
+    }
+
+    @Test
+    public void getTopTracksofArtist_NoContent() throws NoAccountException, ForbiddenException, NoArtistsException, BadRequestException, NotFoundException, UnAuthorizedException {
+        LinkedHashMap<String, ArrayList> inputMap = new LinkedHashMap<>();
+        ResponseEntity responseEntity = new ResponseEntity(inputMap, HttpStatus.NO_CONTENT);
+        Mockito.when(restTemplate.exchange(ArgumentMatchers.anyString(),
+                ArgumentMatchers.any(HttpMethod.class),
+                ArgumentMatchers.<HttpEntity<?>> any(),
+                ArgumentMatchers.<Class<String>> any())).thenReturn(responseEntity);
+        String Message = null;
+        try{
+            topTrackofArtist.getTopTracksofArtist("AZT1010ZZ7H2","1mYsTxnqsietFxj1OgoGbG");
+        }catch (Exception e){
+            Message = e.getMessage();
+        }
+        Assert.assertEquals(AppConstants.NoTrackException_Msg, Message);
+    }
+
+    @Test
+    public void getTopTracksofArtist_Forbidden() {
+        Mockito.when(restTemplate.exchange(ArgumentMatchers.anyString(),
+                ArgumentMatchers.any(HttpMethod.class),
+                ArgumentMatchers.<HttpEntity<?>> any(),
+                ArgumentMatchers.<Class<String>> any())).thenThrow(HttpClientErrorException.Forbidden.class);
+        String Message = null;
+        try{
+            topTrackofArtist.getTopTracksofArtist("AZT1010ZZ7H2","1mYsTxnqsietFxj1OgoGbG");
+        }catch (Exception e){
+            Message = e.getMessage();
+        }
```

Comment:
Formatting.  Do you have a linter?  (You can make a linter part of your PR checks.)